.github/actions/next-internal-version/action.yaml

```diff
@@ -0,0 +1,29 @@
+name: "Get Next Internal Release Version"
```

Comment:
We're referring to these as Cloud releases going forward, right?

Reply:
I would also rename the action itself to match.

Reply:
fair, also update the step name